### DISCUSSION TITLE
Bringing back the dark, Blade Runner style lighting

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#216] [Ideas]
In old alpha screenshots and videos, the global light on levels is much reduced. Instead, there are far more light sources - lamps, windows, even vehicles have head lights illuminating the road.

Compared to that, the final release looks kind of flat. Everything is illuminated, and lights are still there, but it makes little difference if they're disabled.

I wish we could bring back the original tone. I'm pretty sure all levels were originally made with thousands of lights, but at some point there was a decision to use global illumination instead.

Maybe the decision was due to performance, or maybe testers were complaining about some areas being too dark. With global illumination reduced, places where buildings were destroyed would become very dark as soon as the fires fizzle out.

Anyway, I don't think it we're in a position where we could just add lights to the maps and fine-tune them properly. We don't have enough people to do this without introducing problems. Would be much better if we got the old maps from somewhere.. then we could transplant the old lights to new levels. But will that ever happen?

For vehicle head lights - I didn't found any disabled/inactive code for that in the sources. But will keep looking.

**Comment by Moburma** [comment]
I agree this would be nice, the old map overviews show how moody and exciting it looked. That said, I don't see it being practical to really achieve for the reasons you mention. The other option would be to port the improvements from the PlayStation version, but even with my poor grasp of programming that seems like a Herculean task basically akin to reverse engineering an entire other game as well.

I think ultimately this comes down to a bigger issue - growing the community. The more people you have and the more awareness, the more likely you will a) find the right people to work on things and b) get attention from people who have the original game assets. I guess obvious people to ask would be Mike Diskett himself (he has posted fairly recently on the mailing list, I thought of asking him when he did, but I dunno how open to this he'd actually be these days), and the guy who ported the Saturn version. The latter seems to have been an external contractor, so he's likely to have had a copy of the codebase somewhere. Of course it's a lot to ask for anyone to give things to strangers that could potentially get them in legal trouble over a near-forgotten game from 30 years ago, but you never know.

On the community side of things, I thought of maybe making a Discord? I'm not really a big fan of this, but it seems one of the few ways you can go these days. There is a small one for Magic Carpet 2 and it's nice talking to people there, although surprisingly no one there is interested in Syndicate Wars. The other thing would be to make a subReddit. There actually is a Syndicate Wars one, but it's ended up as some guy's personal clubhouse it looks like, as it's private entry with near zero posts. I could ask about taking it over etc. The big elephant in the room is how many people out there would be interested at all. There have been a few on my YouTube videos, but I'm not sure if it would have legs. This is especially true as it's a primarily single player game - the really popular revivals are of course DK and Pop3 as they both have thriving multiplayer. It's hard to keep sustained interest in a game when it's finished after a few days.

**Reply by mefistotelis** [reply]
> On the community side of things, I thought of maybe making a Discord?

I'm not using discord as well. There probably are people who find it easier than Github.
But definitely some prefer Github - in my drone tools project, I see people often talking about an unrelated tool which has discord but no github presence.
